Backend/JourneyLog.DAL/Entities/User.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace JourneyLog.DAL.Entities;
+
+public class User
+{
+    public Guid IdentityId { get; set; }
+    public string FirstName { get; set; }
+    public string LastName { get; set; }
+    public DateOnly DateOfBirth { get; set; }
+
+    public virtual ICollection<List> Lists { get; set; }
```

Comment:
virtual is redundant since we are not using lazy loading

Reply:
Done